cpp/tests/utilities/timestamp_utilities.cuh

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <simt/chrono>
+
+#include <thrust/logical.h>
+#include <thrust/sequence.h>
+#include <thrust/device_vector.h>
+
+namespace cudf {
+namespace test {
+
+using time_point_ms = simt::std::chrono::time_point<
+  simt::std::chrono::steady_clock,
+  simt::std::chrono::milliseconds
+>;
+
+template <typename Rep>
+struct generate_each_timestamp {
+    const Rep base;
+    const Rep step;
+    generate_each_timestamp(Rep _base, Rep _step) : base(_base), step(_step) {}
+    __host__ __device__ Rep operator()(const int32_t& i) const { return base + (step * i); }
+};
+
+/**---------------------------------------------------------------------------*
+ * @brief Creates a ```thrust::device_vector``` with ascending timesamps in the
```

Comment:
I think markdown triple backticks are for code blocks, not for inline code words. Also typo.

```suggestion
 * @brief Creates a `thrust::device_vector` with ascending timestamps in the
```

Reply:
Thanks, updated!